modules/v1/users/models/LoginForm.php

```diff
@@ -64,18 +64,18 @@ public function validatePassword(string $attribute, $params)
         if (!$this->hasErrors()) {
             $user = $this->getActiveUser();
 
-            if (User::find()->byEmailOrUsername($this->username)->notActive()->one()) {
+            if (User::find()->byEmailOrUsername($this->email)->notActive()->one()) {
```

Comment:
@ll-bat Let's change this so that it only searches inside email on inside username